creasee_skywalker.def.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": 2,
+    "name": "Creasee Skywalker",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "manufacturer": "CREASEE",
+        "machine_extruder_trains":
+        {
+            "0": "creasee_extruder_2"
```

Comment:
Why not just call this the "Creasee_skywalker_extruder"? That will make it easier to figure out what extruder belongs where.